lib/rspec/core/configuration.rb

```diff
@@ -1038,6 +1044,7 @@ def reporter
       #
       # Defaults `profile_examples` to 10 examples when `@profile_examples` is
       # `true`.
+      remove_method :profile_examples
```

Comment:
It's quite strange that this `method redefined` warning didn't appear before the changes.

Reply:
Since this would only crash that spec that requires `rspec/core` and the warning is issued, this like satisfies specs and would be useless otherwise.
I went with skipping the definition of the reader for this setter since we override it anyway with a method.

Reply:
Right, I should have removed the reader.
I have no clue why the warning didn't appear before tbh, I didn't investigate.